gotpm/cmd/seal.go

```diff
@@ -125,6 +126,7 @@ func init() {
 	addOutputFlag(unsealCmd)
 	// PCRs and hash algorithm only used for sealing
 	addPCRsFlag(sealCmd)
+	sealCmd.MarkFlagRequired("pcrs")
```

Comment:
We thought requiring PCRs could simplify Seal()/Unseal() . Because sealing to empty PCR is a special case need to be handled separately.
But seems like sealing to empty PCR is an important use case, I can add the logic back.